xml/templates/message_templates/event_online_receipt_html.tpl

```diff
@@ -461,9 +461,11 @@
       {if {event.allow_selfcancelxfer|boolean}}
         <tr>
           <td colspan="2" {$valueStyle}>
-            {capture assign=selfservice_preposition}{if {event.selfcancelxfer_time} > 0}{ts}before{/ts}{else}{ts}after{/ts}{/if}{/capture}
-            {ts 1={event.selfcancelxfer_time} 2=$selfservice_preposition}You may transfer your registration to another participant or cancel your registration up to %1 hours %2 the event.{/ts}
-            {if {contribution.paid_amount|boolean}}{ts}Cancellations are not refundable.{/ts}{/if}<br/>
+            {if {event.selfcancelxfer_time|boolean} && {event.selfcancelxfer_time} > 0}
+              {ts 1="{event.selfcancelxfer_time}"}You may transfer your registration to another participant or cancel your registration up to %1 hours before the event.{/ts}
+            {else}
```

Comment:
@eileenmcnaughton just wondering if this should be elseif with the |boolean option as what if there is no selfcancelxfer time set?

Also we seem to have lost the statement from old like 466

Reply:
@seamuslee001 yeah - I thought that too - I will wait for @mlutfy & @samuelsov to take a look cos they might have further thoughts

on losing that line - I'm a bit confused about that - I was sure I had already removed that line on the basis of ... who are we to say what is refundable!